### DISCUSSION TITLE
Good skull cavern target?

### DISCUSSION THREAD

**Discussion by waynn** [nathan-alden-sr/stardew-valley#30] [General]
What floor or how much iridium is a good Skull Cavern run? I know you have the blank per day calculation but wanted to get a sense, especially with days where you don't start until 930 am since you're upgrading tools. 

Also do you just smelt bars at the beginning of the day and that's it?

@nathan-alden-sr

**Comment by nathan-alden-sr** [comment]
This is a tough one and is highly dependent on many factors.

Early in Year 1, it can be difficult to reach floor 100 even with good luck. Here are some of the biggest factors that affect depth at that time:

- Your Weapon sucks until you get the Galaxy Sword and it's pure RNG whether you get a lucky Prismatic Shard drop
- You will be using Coal constantly to smelt Bars and won't have any to spare for Explosives
- You have the Gold Pickaxe
- Your Pickaxe timing
- You are forced to spend some days--fully or partially--doing other tasks and may miss good luck days because of this

Once Summer rolls around, I found that the extra income from Starfruit Wine combined with the general lack of annoying fetch quests and such to do left me with more time to go to Skull Cavern. In general, if you decide to go, leave as early in the morning as possible.

The real fun begins when you reach the following baseline:

- Enough Diamonds, Emeralds, Jade, and Rubies to stay stocked with Triple Shot Espresso, Cheese, Staircases, and Spicy Eel
- At least three Prismatic Shards per week so you can trade three for a Magic Rock Candy on Thursday
  - Magic Rock Candy is an absolutely _godly_ item and can easily add an extra 100 floors to your run. Using it on a very happy luck day means you'll be exposing ladders with just a few rocks
- A good supply of Warp Totem: Desert items so you can visit The Desert to trade for things like Magic Rock Candy even if you're not running Skull Cavern on that day
- A Lucky Ring and an Iridium Band

As I worked my way into Fall, I was comfortably reaching floors 100-200. One improvement I might be able to make is to use Explosive Ammo instead of Bombs (see the mining guide for info on Explosives). I just find the slingshot annoying, though. I need to practice with it some.

As far as Iridium Ore counts, early in Year 1 you can expect maybe 150 if you're lucky. As the year progresses, expect upwards of 400 or more, especially if you are using Magic Rock Candy. At many points you will have more Iridium Ore than Coal with which to smelt it. Coal eventually becomes _the_ bottleneck unless you don't mind spending tons of G buying it from Clint.

A good rule of thumb is the better your total luck, including daily luck and all buffs, the deeper you should rush before stopping to collect Iridium Ore.

Late in Year 1, you will get more value by spending time on other tasks like a Tree farm, reaching Foraging level 10, etc. Iridium Bars drop off in profitability-to-time as Starfruit Wine rolls around.

**Comment by waynn** [comment]
Did you use crystalariums for emeralds and rubies also? Or just diamonds
and Jade like the original guide?

**Reply by nathan-alden-sr** [reply]
I got lucky and acquired several Crystalariums in chests in Skull Cavern. Early on, I had one Diamond and three Jade. Eventually I added lots more Jade to where between runs I can aquire 100+ Jade. Staircases at this point are essentially infinite given how little I run Skull Cavern.

That said, I find Triple Shot Espresso to actually be pretty damn important, especially in early-Year 1 runs. Until you get the Greenhouse you won't have a reliable way of generating a lot of Coffee Beans to make your own. As the year progresses and you start relying more on Magic Rock Candy, the buff becomes less important.
